packages/notebook/src/actions.tsx

```diff
@@ -803,13 +838,9 @@ export namespace NotebookActions {
 
     const state = Private.getState(notebook);
 
-    notebook.deselectAll();
-    for (let i = notebook.activeCellIndex; i < notebook.widgets.length; ++i) {
-      notebook.select(notebook.widgets[i]);
-    }
-
-    const promise = Private.runSelected(
+    const promise = Private.runCells(
```

Comment:
I have a change in behavior for this one. It does not activate the last cell after it is done (it keeps the selected cell activated).

Reply:
Good catch, thank you! Addressed and tested in 202e958e967955d7a3d94a1d666dd7661a77a644.